schema-dex-amm-extended.graphql

```diff
@@ -652,6 +805,121 @@ type LiquidityPoolHourlySnapshot @entity {
 
   " Per-block reward token emission as of the current block normalized to a day (not hour), in USD value. This should be ideally calculated as the theoretical rate instead of the realized amount. "
   rewardTokenEmissionsUSD: [BigDecimal!]
+
+  ##### Account/Position Data #####
+
+  " Number of positions in this market "
+  positionCount: Int!
+
+  " Number of open positions in this market "
+  openPositionCount: Int!
+
+  " Number of closed positions in this market "
+  closedPositionCount: Int!
+}
+
+####################################
+##### Pool Tick-Level Metadata #####
+####################################
+
+type Tick @entity {
```

Comment:
You are the expert, but why are `tick`s in the `LiquidityPool`s BigInts but we have a tick type?

Reply:
I can see your point here. It does not seem particularly important to have the tick entity for the pool. What mostly matters is what the tick index is so that you can tell if positions are in or out of liquidity. This should suffice for giving relevant information you want when looking at a pool, and should save query speed was my thought. If you are interested in a pools ticks, then you are probably interested in looking at the distribution of ticks and not just one.